jobs/jmeter_storm/templates/run.erb

```diff
@@ -50,7 +51,17 @@ cp $JMX_PLAN_FILE $LOG_DIR/generated_test_plan.jmx
 
 pushd /var/vcap/packages/jmeter/bin
 
+<%
+  user_properties = []
+  if_p('jmeter.reportgenerator.overall_granularity') do
+    user_properties.push("jmeter.reportgenerator.overall_granularity=" << p("jmeter.reportgenerator.overall_granularity").to_s)
+  end
+%>
+
+echo '<%=user_properties.join("\n")%>' > user.properties
+
```

Comment:
@daichi703n 
I am thinking rather than adding the user properties here and only supporting one property, `overall_granularity` in this case, what do you think about having a separate template that renders the `user.properties` file. 

To make it more generic, this template can render an `array` of the these properties, and it is up to the user to specify whatever they want.

For example, having something like this in the job properties:
```yaml
user_properties:
- name: jmeter.reportgenerator.overall_granularity
  value: 30
- name: <some other user property>
  value: ...
``` 

Let me know what do you think